infra/utils.py

```diff
@@ -31,31 +32,31 @@ def chdir_to_root():
     os.chdir(helper.OSSFUZZ_DIR)
 
 
-def is_fuzz_target_local(file_path):
-  """Returns whether |file_path| is a fuzz target binary (local path).
-  Copied from clusterfuzz src/python/bot/fuzzers/utils.py
-  with slight modifications.
-  """
-  filename, file_extension = os.path.splitext(os.path.basename(file_path))
-  if not VALID_TARGET_NAME.match(filename):
-    # Check fuzz target has a valid name (without any special chars).
-    return False
-
-  if file_extension not in ALLOWED_FUZZ_TARGET_EXTENSIONS:
-    # Ignore files with disallowed extensions (to prevent opening e.g. .zips).
-    return False
+def execute(command, location=None, check_result=False):
+  """ Runs a shell command in the specified directory location.
 
-  if not os.path.exists(file_path) or not os.access(file_path, os.X_OK):
-    return False
+  Args:
+    command: The command as a list to be run.
+    location: The directory the command is run in.
+    check_result: Should an exception be thrown on failed command.
 
-  if filename.endswith('_fuzzer'):
-    return True
+  Returns:
+    The stdout of the command, the error code.
 
-  if os.path.exists(file_path) and not stat.S_ISREG(os.stat(file_path).st_mode):
-    return False
+  Raises:
+    RuntimeError: running a command resulted in an error.
+  """
 
-  with open(file_path, 'rb') as file_handle:
-    return file_handle.read().find(FUZZ_TARGET_SEARCH_STRING.encode()) != -1
+  if not location:
+    location = os.getcwd()
+  process = subprocess.Popen(command, stdout=subprocess.PIPE, cwd=location)
+  out, err = process.communicate()
+  if check_result and (process.returncode or err):
```

Comment:
also, once the above is fixed, this condition might quite often turn `True`, even when there was no actual error. Some programs use `stderr` to output auxiliary information. We might want to return both `stdout` and `stderr` from this function